internal/util/indexcgowrapper/index.go

```diff
@@ -28,8 +28,10 @@ type Blob = storage.Blob
 type CodecIndex interface {
 	Build(*Dataset) error
 	Serialize() ([]*Blob, error)
+	SerializeDiskIndex() ([]*Blob, error)
```

Comment:
It seems that the logic of `SerializeDiskIndex` is same to `Serialize`. Any reasons not to directly use `Serialize`?